testing/regress/ecl/redissynctest.ecl

```diff
@@ -177,7 +177,7 @@ END;
 pubDS := DATASET(N2, TRANSFORM({ integer a }, SELF.a := pub('PubSubTest' + (STRING)COUNTER)));
 
 INTEGER pub2(STRING channel) := FUNCTION
-        sl := SEQUENTIAL(
+        sl := ORDERED(
```

Comment:
This change does not seem directly related to this issue.

Reply:
This is to avoid the new warning generated when running the regression suite.